src/components/events-card-new/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { tailwindWrapper } from "../../../../../formula_one/src/utils/tailwindWrapper";
+
+const EventCard = ({ eventName, eventLocation, onRegister, index }) => {
```

Comment:
@Hardik-2311 I think we should use Class based component instead of Function based here, to maintain consistency.

Reply:
ohhkkkk doing it

Reply:
done @beingnoble03